R/delineate.R

```diff
@@ -43,7 +43,7 @@ delineate <- function(
   city_name, river_name, crs = NULL, network_buffer = NULL,
   buildings_buffer = NULL, dem_buffer = 2500, initial_method = "valley",
   initial_buffer = NULL, dem = NULL, max_iterations = 10,
-  capping_method = "shortest-path", angle_threshold = 90, corridor = TRUE,
+  capping_method = "shortest-path", angle_threshold = 100, corridor = TRUE,
```

Comment:
Why has the default angle threshold been increased to 100?

Reply:
Good that you point this out. I noticed results looks slightly better for a higher threshold. I suppose this is because earlier we would have the post-processing step that would still refine the segments, so we could be a little bit more "loose" on what we actually consider as acceptable continuous straight lines.  Right now, the lines generated by rcoins are the lines that are ultimately used as segment boundaries, so to make sure these are "straight enough" we have to apply a higher threshold. Does this make sense to you?

Reply:
Yes, thanks for the explanation!